src/pages/myPage/MypageTable.tsx

```diff
@@ -69,21 +75,31 @@ const MypageTable: React.FC<MypageTableProps> = ({ title, articles }) => {
                   전체보기
                 </Text>
               </Button>
-            </td>
-          </tr>
+            </div>
+          </div>
 
           {articles.length > 0 &&
             articles.map((articleObj, index) => {
               const isLastRow = index === articles.length - 1;
               const borderBottomRadius = isLastRow ? "10px" : "0px";
               return (
-                <tr key={index}>
-                  <td
+                <div
+                  key={index}
+                  style={{
+                    display: "flex",
+                    alignItems: "center",
+                    flexDirection: "row",
+                    backgroundColor: colorSet.colorless,
+                    justifyContent: "space-between",
+                    borderBottomLeftRadius: borderBottomRadius,
+                    borderBottomRightRadius: borderBottomRadius,
+                    border: "0.1px solid #e6e6e6",
+                  }}
+                >
```

Comment:
@dohyun-ko 가 만든 Flex atom을 쓰는 것도 좋아 보입니다
```suggestion
                <Flex
                  key={index}
                  alignItems="center"
                  flexDirection:="row"
                  justifyContent="space-between"
                  style={{
                    backgroundColor: colorSet.colorless,
                    borderBottomLeftRadius: borderBottomRadius,
                    borderBottomRightRadius: borderBottomRadius,
                    border: "0.1px solid #e6e6e6",
                  }}
                >
```